docs/streams/developer-guide.html

```diff
@@ -2245,7 +2256,10 @@
             .groupBy((key, word) -> word, Serialized.with(stringSerde, stringSerde));
 
           // Create a window state store named "CountsWindowStore" that contains the word counts for every minute
-          groupedByWord.count(TimeWindows.of(60000), "CountsWindowStore");
+          groupedByWord.windowedBy(TimeWindows.of(60000))
+            .count(Materialized.&lt;String, Long, WindowStore&lt;Bytes, byte[]&gt;&gt;as("CountsWindowStore")
+                withKeySerde(Serdes.String()); // count() sets value serde to Serdes.Long() automatically
```

Comment:
why do we add this? was the example wrong?

Reply:
as above